tests/middleware.test.js

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai';
+import errorHandler from '../src/middlewares/errorHandler';
+
+describe('Error Handler', () => {
+  let req, res, next;
+  beforeEach(() => {
+    res = {
+      status(code) {
```

Comment:
Nice Mocking express req and res objects

Reply:
Nice work right here!

Reply:
This is amazing work!